vnext/JSI.Desktop.UnitTests/JsiRuntimeUnitTests.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <jsi/jsi.h>
+
+#include <gtest/gtest.h>
+
+#include <functional>
+#include <memory>
+
+namespace facebook {
+namespace jsi {
+
+using RuntimeFactory = std::function<std::unique_ptr<Runtime>()>;
+
+class JsiRuntimeUnitTests : public ::testing::TestWithParam<RuntimeFactory>
+{
+public:
+
+  JsiRuntimeUnitTests() : factory(GetParam()), runtime(factory()), rt(*runtime) {}
+
+  Value eval(const char* code)
+  {
+    return rt.global().getPropertyAsFunction(rt, "eval").call(rt, code);
+  }
+
+  Function function(const std::string& code)
+  {
+    return eval(("(" + code + ")").c_str()).getObject(rt).getFunction(rt);
+  }
+
+  bool checkValue(const Value& value, const std::string& jsValue)
+  {
+    // TODO (yicyao): Should we use === instead of == here?
```

Comment:
>=== [](start = 36, length = 3)

yes, i believe in javascript, there is no reason to use ==, just use === everywhere

Reply:
The usage of "==" is coming from Facebook upstream. I'll chat with you offline about this.